bundles/org.openhab.binding.ihc/src/main/java/org/openhab/binding/ihc/internal/handler/IhcHandler.java

```diff
@@ -185,6 +185,9 @@ public void initialize() {
             logger.debug("Start control task, interval={}sec", 1);
             controlJob = scheduler.scheduleWithFixedDelay(this::reconnectCheck, 0, 1, TimeUnit.SECONDS);
         }
+
+        updateStatus(ThingStatus.UNKNOWN, ThingStatusDetail.NONE,
```

Comment:
This should happen before starting the job to avoid any concurrency/timing related issues.

Reply:
Yes, you are right...fixed